webpack/JobWizard/steps/HostsAndInputs/SelectGQL.js

```diff
@@ -35,7 +36,7 @@ export const useNameSearchGQL = apiKey => {
       subtotal: data?.[dataName[apiKey]]?.totalCount,
       results:
         data?.[dataName[apiKey]]?.nodes.map(node => ({
-          id: node.name,
+          id: decodeId(node.id),
```

Comment:
@MariaAga does `host.id` being an actual id instead of a name have potential to break something somewhere?

Reply:
Not from what I see